source/dataman/export/AMExporterOptionGeneralAscii.h

```diff
@@ -60,16 +60,12 @@ class AMExporterOptionGeneralAscii : public AMExporterOptionGeneral
 	/// Sets a precision for a source name. Duplicate naming is not allowed.
 	void setExportPrecision(const QString& source, const int precision);
 	/// Sets the default precision with no AMDataSource name specified.
-	void setExportPrecision(const int precision);
-	/// Loads the precision map into the database in the form of a string.
-	QString loadPrecisionMap();
-	/// Parses a precision map string from the database and recreates the map.
-	void readPrecisionMap(const QString& stringMap);
+	void setExportPrecision(int precision);
 
 	virtual QWidget* createEditorWidget();
 
 	/// Output the current map in exportPrecision_ into a string to allow mapping visualization.
-	QString mapToString();
+	const QString mapToString();
```

Comment:
Yes, because you would be using `at()` which is a `const` function.  See!! Tying it all together! :)